unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Tests/LandElementViewShould.cs

```diff
@@ -25,10 +25,12 @@ public void TriggerCallbacks()
             bool triggered = false;
 
             void OnCallback(string value) { triggered = true; }
+            void OnCallbackVector2(Vector2Int value) { triggered = true; }
 
-            view.OnEditorPressed += OnCallback;
+            view.OnEditorPressed += OnCallbackVector2;
             view.OnSettingsPressed += OnCallback;
-            view.OnJumpInPressed += OnCallback;
+            view.OnOpenInDappPressed += OnCallback;
+            view.OnJumpInPressed += OnCallbackVector2;
```

Comment:
Maybe we could use more descriptive names here ( for `OnCallbackVector2` and `OnCallback`)

Reply:
I'm open to suggestions, I can't think in more descriptive names for this callbacks in this test. We are interested in knowing if the callbacks are called in a test called `LandElementViewShould.TriggerCallbacks` 🤷

Reply:
Ups sorry, I didn't realise it was test code. Forget my message! :)